src/clockid.rs

```diff
@@ -94,6 +95,51 @@ pub enum ClockId {
     BoottimeAlarm = bitcast!(c::CLOCK_BOOTTIME_ALARM),
 }
 
+#[cfg(not(any(apple, target_os = "wasi")))]
```

Comment:
I thought this would be straightforward, but it's really not. libc has `clockid_t`, while linux-raw-sys has `__kernel_clockid_t`. Some platforms have neither. What's the best way to proceed with this? I need it for `get_txtime`, since the kernel returns the configured clock.